### ISSUE TITLE
Bug fix in Activiti 6 for issue #1902

### ISSUE THREAD

**Issue by msjhall138** [Activiti/Activiti#2536]
Hi,

I saw recently that the PR for a new version of Activiti 6 was closed because of too many changes in it: https://github.com/Activiti/Activiti/pull/1899

This PR https://github.com/Activiti/Activiti/pull/1912 included a bug fix for this issue https://github.com/Activiti/Activiti/issues/1902 . 

Is it possible we could reopen a PR for a new version of Activiti 6.x which just includes this fix?

Thanks

**Comment by salaboy** [comment]
@msjhall138 yes.. we can open a PR with that fix only. It was quite difficult for us to do that from the previous PR that contained 913 file changed. 

Also @daisuke-yoshimoto PR was merged already.

**Comment by msjhall138** [comment]
Cheers @salaboy! Yeah I had only linked the PR to reference the fix that I was hoping for in a new version of the 6.x branch.

**Comment by salaboy** [comment]
@msjhall138 I am not in charge of that release train, but let's see if @balsarori can provide some estimates.

**Comment by csabapocsai** [comment]
Hi!

When this fix will be available for public usage? We would like to use Spring Activiti with Spring Boot. 
Could you please provide us some estimates?

Regards,
Csaba

**Comment by salaboy** [comment]
@csabapocsai hi there.. we are putting all the efforts into the 7.x series, you can track the progress here: https://github.com/Activiti/Activiti/milestones , @balsarori might be able to provide more info about the 6.x branch, but as far as I am aware there is no estimate for a new release on that branch.

**Comment by csabapocsai** [comment]
Thank you for replying. In which 7.x version can we use for getting this fix? And when it will be available in a starter project called activiti-spring-boot-starter-basic? As I see the latest is 6.0.0, is there any plan to update it as well in the near future?

**Comment by salaboy** [comment]
@csabapocsai ok.. so what we can do is to forward port the fix into 7.1.x

**Comment by salaboy** [comment]
@balsarori @daisuke-yoshimoto can you help me to understand what changes were merged in 6.x as part of this issue? and create a PR for 7.x

**Comment by csabapocsai** [comment]
> @csabapocsai ok.. so what we can do is to forward port the fix into 7.1.x

My core question is when this fix will be available publicly. Do you have any plan for it?

**Comment by salaboy** [comment]
@csabapocsai yes.. if the fix can be forward ported to 7.1.x we should be able to release it by the end of March in 7.1.0.M1
